form.js

```diff
@@ -119,6 +120,17 @@ define('form', ['doc'], function($) {
 			}
 		});
 
+		form.find('[type="url"]').each(function(element){
+			var pattern = new RegExp(/^http(s)?:\/\/(www\.)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/),
```

Comment:
I tested it here on Google Chrome and it works. The fact is: we put the `novalidate` attribute on the form, but we can verify validity at field level (using the expression I mentioned above).

@luiz